### PR TITLE
Add ref_numbers to containers

### PR DESCRIPTION
**Pull request by edymerchk**

### CODE REVIEW

File: docs/reference/terminal49/terminal49.v1.json [hunk 1]
```diff
@@ -5359,6 +5391,13 @@
               "number": {
                 "type": "string"
               },
+              "ref_numbers": {
+                "type": "array",
+                "items": {
+                  "type": "string"
+                },
+                "nullable": true
```
Comment (mattyturner) [inline docs/reference/terminal49/terminal49.v1.json:5399]: Does the serializer return null or an empty array if it's not there?

**Review by mattyturner — COMMENTED** [review]
We need to add the patch endpoint for adding ref_numbers to a container through the API.

File: docs/reference/terminal49/terminal49.v1.json (hunk 1 above)
Comment (edymerchk) [reply docs/reference/terminal49/terminal49.v1.json:5456]: fixed ace39829c980c4a2cbbe3110c49998a75ed77156

Comment (edymerchk) [reply docs/reference/terminal49/terminal49.v1.json]: fixed b6a23815cfb07525018946306c58ed7435242269

**Pull request merged**